plugin/kubernetes/object/object.go

```diff
@@ -16,14 +16,18 @@
 package object
 
 import (
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+        "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
maybe run a `gofmt` to clean it up this kind of thing...

Reply:
Formatted.